src/content/docs/mobile-monitoring/mobile-monitoring-ui/application-not-responding.mdx

```diff
@@ -23,11 +23,10 @@ import mobileAnrSettingToggle from 'images/mobile_screenshot-crop_anr-setting-to
 
 In Android mobile apps, [Application Not Responding (ANR) errors](https://developer.android.com/topic/performance/vitals/anr) are triggered when the main UI thread of an application is blocked for more than five seconds. ANR errors can be caused by various factors, such as network calls, complex calculations, or resource contention. By monitoring their frequency and occurrence patterns, you gain valuable insights into your app's overall stability and user experience.
 
-## How ANR errors are reported in New Relic [#anr-reporting]
+## How ANR errors are reported to New Relic [#anr-reporting]
 
-Our Android agent reports ANR errors as [`MobileApplicationExit`](/attribute-dictionary/?event=mobileApplicationExit) events, which leverage Android’s [`ApplicationExitInfo` API](https://developer.android.com/reference/android/app/ApplicationExitInfo) to monitor and report on the different causes for app termination. The agent reads the `ApplicationExitInfo` API on the next app launch and asynchronously sends `MobileApplicationExit` events to New Relic for each app exit in the history.
+Our Android agent reports ANR errors as [`MobileApplicationExit`](/attribute-dictionary/?event=MobileApplicationExit) events, which leverage Android’s [`ApplicationExitInfo` API](https://developer.android.com/reference/android/app/ApplicationExitInfo) to monitor and report on the different causes for app termination. The agent reads the `ApplicationExitInfo` API on the next app launch and asynchronously sends `MobileApplicationExit` events to New Relic for each app exit in the history.
```

Comment:
Should we add a note here: that we report ANR errors by default, if they would like to disable look at the Disable ANR monitoring section below